### PR TITLE
Index cell_line fields (#930)

### PR DESCRIPTION
**Pull request by danielsotirhos**

### CODE REVIEW

File: test/indexer/test_hca_indexer.py [hunk 1]
```diff
@@ -651,10 +652,22 @@ def test_cell_line_sample(self):
...
+                    self.assertEqual(one(contents['specimens'])['organ'], ['blood'])
+                    self.assertEqual(one(contents['specimens'])['organ_part'], ['venous blood'])
+                    self.assertEqual(one(contents['cell_lines'])['model_organ'], ['blood (cell_line)'])
+                    self.assertEqual(one(contents['cell_suspensions'])['organ'], ['blood (cell_line)'])
+                    self.assertEqual(one(contents['cell_suspensions'])['organ_part'], [None])
                 else:
                     document_ids = [d['document_id'] for d in contents[sample_entity_type]]
                     entity = one([d for d in contents[sample_entity_type] if d['document_id'] == sample['document_id']])
                     self.assertEqual('cell_line_Day7_hiPSC-CM_BioRep2', entity['biomaterial_id'])
+                    self.assertEqual(one(contents['specimens'])['organ'], 'blood')
```
Comment (hannes-ucsc) [inline test/indexer/test_hca_indexer.py]: Almost the same as the line in the if branch. Only difference is that RHS of comparison is `["…"]` vs `"…"`

File: test/indexer/test_hca_indexer.py [hunk 2]
```diff
@@ -651,10 +652,22 @@ def test_cell_line_sample(self):
...
+                    self.assertEqual(one(contents['specimens'])['organ_part'], ['venous blood'])
```
Comment (hannes-ucsc) [inline test/indexer/test_hca_indexer.py]: Identical.

File: test/indexer/test_hca_indexer.py [hunk 3]
```diff
@@ -651,10 +652,22 @@ def test_cell_line_sample(self):
...
+                    self.assertEqual(len(contents['cell_lines']), 2)
```
Comment (hannes-ucsc) [inline test/indexer/test_hca_indexer.py]: Similar, RHS is 1 or 2 in either branch.

File: test/indexer/test_hca_indexer.py [hunk 4]
```diff
@@ -651,10 +652,22 @@ def test_cell_line_sample(self):
...
+                    cell_lines_model_organ = {cl['model_organ'] for cl in contents['cell_lines']}
+                    self.assertEqual(cell_lines_model_organ, {'blood (cell_line)'})
+                    self.assertEqual(one(contents['cell_suspensions'])['organ'], ['blood (cell_line)'])
```
Comment (hannes-ucsc) [inline test/indexer/test_hca_indexer.py]: Identical

File: test/indexer/test_hca_indexer.py [hunk 5]
```diff
@@ -651,10 +652,22 @@ def test_cell_line_sample(self):
...
+                    self.assertEqual(one(contents['cell_suspensions'])['organ_part'], [None])
```
Comment (hannes-ucsc) [inline test/indexer/test_hca_indexer.py]: Identical.

File: src/azul/service/responseobjects/hca_response_v5.py [hunk 6]
```diff
@@ -725,6 +725,8 @@ def make_cell_suspensions(self, entry):
     def make_cell_line(self, cell_line):
         return {
             "id": cell_line["biomaterial_id"],
+            "cellLineType": cell_line.get("cell_line_type", None),
```
Comment (hannes-ucsc) [inline src/azul/service/responseobjects/hca_response_v5.py]: I'm surprised that these two additional keys don't require some change in a service test assertion.

File: test/indexer/test_hca_indexer.py (hunk 1 above)
Comment (danielsotirhos) [reply test/indexer/test_hca_indexer.py]: Refactored test case to minimize duplication

File: src/azul/service/responseobjects/hca_response_v5.py (hunk 6 above)
Comment (danielsotirhos) [reply src/azul/service/responseobjects/hca_response_v5.py]: Thought this was covered by existing test cases however turns out they are checking the response from projects with no cell_lines. Added a new test case to fix this missing coverage.

**Pull request merged**